server/api/email_handler.py

```diff
@@ -0,0 +1,29 @@
+from flask import jsonify, Blueprint
+from flask import Flask, request, json, jsonify, make_response
+from database.models import user, receipt, picture
+from config import SENDGRID_API_KEY
+import os
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+email_handler = Blueprint('email_handler', __name__)
+
+
+@email_handler.route('/sendEmail', methods=['GET'])
+def sendEmail():
+
+    message = Mail(
+        from_email='teamravioli6@gmail.com',
+        to_emails='teamravioli6@gmail.com',
```

Comment:
change this function to take in an email string so that it's ready to send an email to that user

Reply:
I was thinking to add that in the next PR when I pull user info from auth_token.